### ISSUE TITLE
Account creation: update copies for WooCommerce

### ISSUE THREAD

**Issue by chihsuan** [Automattic/wp-calypso#74148]
p2: peapX7-1Sn-p2
figma: lD0gTB4IqGsmSmsnjZNaP6-fi-1%3A144

Copy changes for WooCommerce account creation.

### Account step – New user

<img width="1755" alt="account-new-user" src="https://user-images.githubusercontent.com/4344253/223325336-ec3d80d9-c1ad-43f3-a7cc-10faebec67d2.png">

1. Change the heading from `Get started in minutes` to `Let's get started.`
2. Change the description from `First, let's create your account. Already registered? Log in` to `All Woo stores are powered by WordPress.com. Please create an account to continue. Already registered? Log in.` 

### Account step – Existing WP.com user, logged in

<img width="1755" alt="account-existing-user-logged-in-1" src="https://user-images.githubusercontent.com/4344253/223325131-e67207c7-b5d6-472a-a341-bb3a9123f4f3.png">

1. Change the heading from `Get started in minutes` to `Let's get started`.
2. Change the description from `First, log in with your WordPress.com account. Don't have an account? Sign up` to `All Woo stores are powered by WordPress.com! First, select the account you'd like to use.`

### Account step – Existing WP.com user, logged out

<img width="1755" alt="account-existing-user-logged-out-1" src="https://user-images.githubusercontent.com/4344253/223325215-012d7215-8c88-467c-b00d-9865a8155ff2.png">

1. Change the heading from `Get started in minutes` to `Let's get started`.
2. Change the description from `First, select the account you'd like to use.` to `All Woo stores are powered by WordPress.com! Please, log in to continue. Don't have an account? Sign up.`

Also, we need to update the button and link colors.
<img width="1440" alt="Account - Signup" src="https://user-images.githubusercontent.com/4344253/228425484-b7c89e62-0ec0-41ab-857f-e0eb6f0ba9da.png">

**Comment by arjunpotter17** [comment]
Hey, I've fixed the issue and made a pull request. Please help me out.

**Comment by arjunpotter17** [comment]
@chihsuan @adrianduffell  can you help me out with the process? is there anything else that I have to do?

**Comment by chihsuan** [comment]
Hey @arjunpotter17, Thanks for updating it! Could you add some testing instructions to your PR? (refer to https://github.com/Automattic/wp-calypso/pull/71178.) Thank you! We'll review it soon!

**Comment by arjunpotter17** [comment]
@chihsuan I've added testing instructions. I might be wrong though, it's a little hard to convey changes made in the editor to the browser. On a different note, I have set up calypso locally using the calypso bootstrap version. It is running fine on my localhost, but there are only two folders on my code editor (VS Code) - a .vagrant folder and a .puppet folder. How do I edit code to reflect it on my browser? I previously did it using the inspect element but I cannot rely on the inspect element for more complex bugs. Any suggestions?
